app/models/message.rb

```diff
@@ -1,12 +1,11 @@
 class Greeting < ApplicationRecord
```

Comment:
- [ ] As requsted, please change the file name from 'message.rb' to 'greeting.rb' as rails has the naming conventions and could't find any greeting model as you don't define it correctly.